portfolio/src/main/webapp/script.js

```diff
@@ -51,23 +51,45 @@ async function showSongLyrics() {
 
 /** Script for time */
 function GetClock(){
```

Comment:
I am not sure about this. Can you explain what exactly are you trying to do here? Also
- You don't need 2 functions & should probably combine them
- I dont see where getClock is being called
- Keep the function name 'getClock' instead that is a more common function name pattern where it starts with a smaller case.